src/components/table/expandable/row/style.css

```diff
@@ -35,18 +35,18 @@ td::slotted(*) {
 	--expansion-width: 1.5rem;
 	--expansion-border-width: 3px;
 	position: relative;
-	background-color: rgb(var(--smoothly-default-color));
+	background-color: rgb(var(--smoothly-color));
 	width: calc(100% + 3rem - 2px);
 	left: calc(-1 * var(--expansion-width) + 1px);
 	box-sizing: border-box;
 	padding: 0.5rem calc(var(--expansion-width) - 1px);
-	border-bottom: 1px solid rgb(var(--smoothly-light-shade));
+	border-bottom: 1px solid rgb(var(--smoothly-color-shade));
 }
 tr.spotlight > td::slotted(*) {
-	box-shadow: calc(var(--expansion-border-width)) 0px 0px 0px rgb(var(--smoothly-tertiary-color)) inset;
+	box-shadow: calc(var(--expansion-border-width)) 0px 0px 0px rgb(var(--smoothly-color-shade)) inset;
```

Comment:
This color should stand out from the rest. this is the left side line. This is a color that maybe should override this pattern and maybe be --smoothly-secodary-color or --smoothly-tertiary-color. It should not have the same color as the rest of the table.

Reply:
Makes sense, yes!